GVFS/GVFS.Virtualization/Projection/GitIndexProjection.FolderEntryData.cs

```diff
@@ -9,6 +9,12 @@ internal abstract class FolderEntryData
         {
             public LazyUTF8String Name { get; protected set; }
             public abstract bool IsFolder { get; }
+            public bool IsIncluded { get; set; } = true;

```

Comment:
It has to be in this base class and not just in the `FileData` because the code also has to know if a folder is included or not.  Measuring the memory and for ~3 million entries in the index it increased the memory usage ~20 MB from ~710 MB to ~730 MB.  So doesn't seem like a big enough amount to change it.  What do you think?

Reply:
> It has to be in this base class and not just in the FileData because the code also has to know if a folder is included or not

Sorry it looks like I left a half thought.  If we did decide to try to steal a bit for files we'd need to make `IsIncluded` abstract and add the bool to the folder data.

> Measuring the memory and for ~3 million entries in the index it increased the memory usage ~20 MB from ~710 MB to ~730 MB. So doesn't seem like a big enough amount to change it. What do you think?

I agree that doesn't sound bad.  I think we can stick with the current code, I'll mark this comment as resolved.